src/Octoshift/GithubApi.cs

```diff
@@ -138,7 +138,7 @@ public virtual async Task<string> CreateGhecMigrationSource(string orgId, string
                 variables = new
                 {
                     name = "GHEC Source",
-                    url = "https://github.com",
+                    url = sourceUrl,
                     ownerId = orgId,
                     type = "GITHUB_ARCHIVE",
```

Comment:
We are getting migration type is not supported from GHES and had to update type = "GITHUB".
"Git source migration failed. Error message: The migration source url is not supported for this migration type."